src/bin/wapm.rs

```diff
@@ -1,20 +1,25 @@
+#![cfg_attr(target_os = "wasi", allow(unused_variables))]
 use std::{env, path};
 use structopt::{clap::AppSettings, StructOpt};
 #[cfg(feature = "update-notifications")]
 use wapm_cli::update_notifier;
+#[allow(unused_imports)]
 use wapm_cli::{commands, logging};
 
 #[derive(StructOpt, Debug)]
 #[structopt(global_settings = &[AppSettings::VersionlessSubcommands, AppSettings::ColorAuto, AppSettings::ColoredHelp])]
 enum Command {
+    #[cfg(feature = "full")]
     #[structopt(name = "whoami")]
     /// Prints the current user (if authed) in the stdout
     WhoAmI,
 
+    #[cfg(feature = "full")]
     #[structopt(name = "login")]
     /// Logins into wapm, saving the token locally for future commands
     Login,
 
+    #[cfg(feature = "full")]
```

Comment:
Can we re-enable login, logout and whoami? It will be quite interesting to plug those automatically so an user is automatically logged in to their wapm user!

Reply:
yeah I can have a go at this - there is a bunch of functionality I had to disable because of the linkage to a local SQL engine that needed dependencies not in WASM

What I'll probably also do is store it in the /private folder so that other processes cant use the token without going via WAPM - basically thats the same as the tok commands

Reply:
Login, Logout and Whoami are now enabled.

Note: The token is stored in /.private which is only visible to the running process so you can not see the token even though you have logged on - this is a security feature to stop other processes stealing the token